google-cloud-examples/src/main/java/com/google/cloud/examples/pubsub/snippets/CreateSubscriptionAndConsumeMessages.java

```diff
@@ -31,7 +30,7 @@
  * A snippet for Google Cloud Pub/Sub showing how to create a Pub/Sub pull subscription and
  * asynchronously pull messages from it.
  */
-public class CreateSubscriptionAndPullMessages {
+public class CreateSubscriptionAndConsumeMessages {
```

Comment:
There is a reference to it in `google-cloud-pubsub/README.md`. Could you please modify there too?